src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -109,7 +112,7 @@ pushsource* createPushSource(uint32 srcid, const char* name) {
 }
 
 void OnCheck(uv_check_t* handle) {
-	const uv_loop_t* const loop = handle->loop;
+    const uv_loop_t* const loop = handle->loop;
```

Comment:
Indent error.  You seem to be using spaces whereas everything else in this file uses tabs.